docs/ff-concepts/navigation-routing/passing-data.md

```diff
@@ -0,0 +1,162 @@
+---
+slug: passing-data
+title: Passing Data
+tags: []
+description: Learn how to pass data between pages in FlutterFlow.
+sidebar_position: 2
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Passing Data between Pages
+
+As you build your app, you'll often encounter the need to pass through or transfer data from one page to another. For instance, when a user taps on a product item, you may want to send product data to the next page to display its details.
+
+## How to pass data
```

Comment:
Instead of a question, we should name it the "Page parameters" header because its easier to find via SEO (people will mostly search for page parameters when they encounter it in FF) and also straightaway defines what the section is about.

Reply:
Done!